client/components/wcc-settings-form/settings-group.js

```diff
@@ -43,13 +49,20 @@ const SettingsGroup = ( { group, schema, storeOptions, settings, form, formActio
 					/>
 				</CompactCard>
 			);
+
+		default:
+			return (
+				<div className="settings-group-default">
+					{ group.items ? renderGroupItems( group.items, schema, storeOptions ) : null }
```

Comment:
Yeah, the lack of recursion between this and settings-item is giving me a twitch. I might need to do something about that.